ui/user-details/UserDetails.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import { motion } from "framer-motion"
+import { Box, Heading, Avatar } from "grommet"
+import { User } from "grommet-icons"
+import { UserInfo } from "../../types/UserInfo"
+
+interface UserDetailsProps {
+    user: UserInfo
+    small?: boolean
+}
+export const UserDetails: React.FunctionComponent<UserDetailsProps> = (props) => {
+    const { user, small } = props
+
+    if (small) {
```

Comment:
why not just conditionally set size inline instead of doubling the code with these two conditional returns?

Reply:
like just do size={small ? 'medium' : 'large'}

Reply:
this one doesn't have the user name, its the small mobile one